index.js

```diff
@@ -134,7 +134,7 @@ const airtableIntervalMs = parseInt(process.env.AIRTABLE_SYNC || 0);
 if (airtableIntervalMs > 0) {
   airtableWorker(airtableIntervalMs);
   if (process.env.AIRTABLE_PAYMENTS_BASE) {
-    airtablePaymentsWorker(airtableIntervalMs);
+    airtablePaymentsWorker(airtableIntervalMs + 2000); // Stagger polling to avoid rate limit
```

Comment:
I was thinking if this becomes a serious problem we can make something to automatically enforce the rate limit. We could make a facade around the instance that comes out of `airtable("tableName")` that pauses before `select`/`update`/ etc if needed.

Reply:
Definitely a good idea